hummingbot/strategy/spot_perpetual_arbitrage/spot_perpetual_arbitrage.py

```diff
@@ -237,22 +236,22 @@ async def create_base_proposals(self) -> List[ArbProposal]:
         :return: A list of 2 base proposals.
         """
         tasks = [self._spot_market_info.market.get_order_price(self._spot_market_info.trading_pair, True,
-                                                               self._order_amount),
+                                                               self._max_possible_arbitrage_amount),
                  self._spot_market_info.market.get_order_price(self._spot_market_info.trading_pair, False,
-                                                               self._order_amount),
+                                                               self._max_possible_arbitrage_amount),
                  self._perp_market_info.market.get_order_price(self._perp_market_info.trading_pair, True,
-                                                               self._order_amount),
+                                                               self._max_possible_arbitrage_amount),
                  self._perp_market_info.market.get_order_price(self._perp_market_info.trading_pair, False,
-                                                               self._order_amount)]
+                                                               self._max_possible_arbitrage_amount)]
```

Comment:
`_market_info.market.get_order_price()` has not been changed, lets assume it returns ob price levels up till `_max_possible_arbitrage_amount`. list of [order_price, order_amount]